assets/js/scrollDepth.js

```diff
@@ -0,0 +1,46 @@
+const SCROLL_DEPTH_LEVELS = [20, 40, 60, 80, 99];
+
+window.addEventListener('DOMContentLoaded', function () {
+  let observer
+  const el = document.getElementsByTagName('body')[0];
+    
+  el.style.position = 'relative';
+
+  const initObserver = () => {
+    const cb = (entries) => {
+    const [IntersectionObserverEntry] = entries;
+    const { isIntersecting, target } = IntersectionObserverEntry;
```

Comment:
Do it in args I prefer, it simpler I think
`( [ { isIntersecting, target } ] ) =>`

Reply:
It is simpler but not better for readability IMO. but I can destructure it.